### PR TITLE
Google forms integration

### PR DESCRIPTION
**Pull request by amohamed97**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -42,5 +40,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```
Comment (shakram02) [inline public/index.html]: new line

File: src/api-client.js [hunk 2]
```diff
@@ -8,10 +8,30 @@ export default {
   getLabs() {
     return axios.get(`${graderx_api}labs/cc451`);
   },
+
   uploadSubmissions(labId, formData) {
     return axios.post(`http://localhost:5000/results/cc451/${labId}`, formData);
```
Comment (shakram02) [inline src/api-client.js]: URLS should go in config variables/env variables

File: src/components/grader/submissions/ImportSubmissions.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                    }
+
+                </Container>
+            </Transition>
+        </React.Fragment>
+    )
+}
+
+
+export default ImportSubmissions
```
Comment (shakram02) [inline src/components/grader/submissions/ImportSubmissions.js]: new line

**Review by shakram02 — CHANGES_REQUESTED** [review]

File: src/api-client.js [hunk 4]
```diff
@@ -10,28 +10,28 @@ export default {
   },
 
   uploadSubmissions(labId, formData) {
-    return axios.post(`http://localhost:5000/results/cc451/${labId}`, formData);
+    return axios.post(`${graderx_api}results/cc451/${labId}`, formData);
```
Comment (shakram02) [inline src/api-client.js]: I think it should be `${GRADERX_API}/` note the trailing slash. just to make it very clear it's a base URL

**Review by shakram02 — CHANGES_REQUESTED** [review]

File: package.json [hunk 5]
```diff
@@ -11,8 +11,11 @@
     "js-file-download": "^0.4.12",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-google-login": "^5.1.21",
```
Comment (Hagar-Usama) [inline package.json:14]: Do we implement it in FE?!

**Review by Hagar-Usama — DISMISSED** [review]

**Review by shakram02 — APPROVED** [review]

**Review by Hagar-Usama — APPROVED** [review]

**Pull request merged**